src/libraries/System.IO.MemoryMappedFiles/src/System/IO/MemoryMappedFiles/MemoryMappedFile.cs

```diff
@@ -515,5 +390,94 @@ private static void CleanupFile(FileStream fileStream, bool existed, string path
                 File.Delete(path);
             }
         }
+
+        private static void ValidateParametersCreateFromFile(FileStream fileStream, string? mapName, long capacity,
```

Comment:
I created 3 different parameter validation methods because they are not exactly equal, and I didn't want to introduce breaking changes by either changing the type of exception thrown for certain cases (like the mapName null check), or by changing the order. Thoughts?

Reply:
Not necessary anymore.